frontend/src/components/WaitingRoom/MenuDevices/MenuDevices.tsx

```diff
@@ -1,10 +1,12 @@
-import { Menu, MenuItem } from '@mui/material';
-import { Speaker } from '@mui/icons-material';
 import { AudioOutputDevice, Device } from '@vonage/client-sdk-video';
 import { ReactElement, useMemo } from 'react';
-import { isGetActiveAudioOutputDeviceSupported } from '../../../utils/util';
-import SoundTest from '../../SoundTest';
+import MenuItem from '@ui/MenuItem';
+import Menu from '@ui/Menu';
+import VividIcon from '@components/VividIcon';
+import Box from '@ui/Box';
 import cleanAndDedupeDeviceLabels from '../../../utils/cleanAndDedupeDeviceLabels/cleanAndDedupeDeviceLabels';
+import SoundTest from '../../SoundTest';
+import { isGetActiveAudioOutputDeviceSupported } from '../../../utils/util';
```

Comment:
lets use the aliases whenever is possible 
```suggestion
import { isGetActiveAudioOutputDeviceSupported } from '@utils/util';
```

Reply:
Fixed everywhere